bson/json_util.py

```diff
@@ -467,6 +469,11 @@ def dumps(obj: Any, *args: Any, **kwargs: Any) -> str:
        Accepts optional parameter `json_options`. See :class:`JSONOptions`.
     """
     json_options = kwargs.pop("json_options", DEFAULT_JSON_OPTIONS)
+    if "max_length" in kwargs:
```

Comment:
What do think about calling `truncate_documents` explicitly in the logging code instead of here? That way we don't need to add or document a new "max_length" feature which would only be used internally.

Reply:
Good idea, thanks!